src/components/MaterialsDropdown/MaterialsDropdown.js

```diff
@@ -0,0 +1,33 @@
+import { gql, useQuery } from '@apollo/client'
+import Select from 'react-select';
+
+const MaterialsDropdown = ({ updateMaterialId }) => {
+
```

Comment:
Glad you made a separate component for MaterialsDropdown, Anthony! This makes a lot of sense.